File: RestApi/model/book.js
var author = require("./author")
var shipping = require("./shipping")

class Book{
    constructor(){
        this.bookId = null;
        this.ISBN = null;
        this.title = null;
        this.cover = null;
        this.author = new author();
        this.pusblisher = null;
        this.description = null;
        this.pages = 0;
        this.format = null;
        this.language = null
        this.price = 0;
        this.shipping = new shipping();
    }
}
module.exports = Book;

File: Android/app/src/main/java/model/Book.java
package model;

import java.util.List;

public class Book {
    public String bookId;
    public String ISBN;
    public String title;
    public String cover;
    public Author author;
    public String pusblisher;
    public String description;
    public String language;
    public Shipping shipping;
    public double price;
    public int pages;
    public String format;
    public Book(){
         shipping = new Shipping();
         author = new Author();
    }
}